dev-env.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+## Run docker Compose
+sudo docker compose --env-file backend/.env up -d
+
+## change directory to backend
+cd backend
+
+## start the backend development server
+npm run start:dev &
+
+## change directory to frontend
+cd ../frontend
+
+##  start the frontend development server
+npm run dev & gnome-terminal -- npm run dev
```

Comment:
Here `npm run dev` is executed two times which cause the frontend server to start on both port 3000 and 3001.